examples/dev/package.json

```diff
@@ -1,16 +1,18 @@
 {
+  "name": "uppy-dev-example",
+  "aliasify": {
+    "aliases": {
+      "@uppy": "../../packages/@uppy"
+    }
+  },
   "dependencies": {
-    "@babel/core": "^7.2.2",
+    "@babel/core": "^7.4.4",
     "aliasify": "^2.1.0",
     "babelify": "^10.0.0",
     "watchify": "^3.11.0"
   },
+  "private": true,
   "scripts": {
-    "watch:sandbox": "watchify -t babelify -t aliasify main.js -o bundle.js -vd"
-  },
-  "aliasify": {
-    "aliases": {
-      "@uppy": "../../packages/@uppy"
-    }
+    "watch:sandbox": "watchify -vd -t [ babelify --cwd ../../ ] -g aliasify main.js -o bundle.js"
```

Comment:
:+1: But why did it work before? Something changed in that regard in Babel 7?

Reply:
yeah, there was a change to accomodate npm linked packages I think, now only things inside the `cwd` are processed by babel. browserify already handles that, but babel now has its own filtering, so now babelify can ignore files that browserify gives it.